tutorial/NECost-OT.son

```diff
@@ -0,0 +1,69 @@
+necost {
+    construction_interest_rate = 0.05
```

Comment:
do we have anywhere description of this input in the documentation? Or is this next step?

Reply:
Next step